src/App.js

```diff
@@ -21,7 +21,7 @@ function App(props) {
       ) : null}
       <Switch>
         <Route exact path="/" component={Landing} />
-        <Route exact path="/register" component={Register} />
+        <Route path="/(login|register)/" exact component={Auth} />
```

Comment:
Great job figuring this out. I learnt something new today

Reply:
Thank you